TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/default_operations.py

```diff
@@ -201,6 +201,48 @@ def get_noop_transformation(self):
         return {
             "NoopTransformerProvider": ""
         }
-    
+
     def run_test_benchmarks(self, cluster: Cluster):
         run_test_benchmarks(cluster=cluster)
+
+    def disable_bloom(self, cluster: Cluster, index_name: str):
+        """
+        Disable bloom_filter_for_id_field for a given index (ES 8.x specific).
+        """
+        path = f"/{index_name}/_settings"
+        headers = {'Content-Type': 'application/json'}
+        payload = {
+            "index": {
+                "bloom_filter_for_id_field": {
+                    "enabled": False
+                }
+            }
+        }
+
+        response = execute_api_call(
+            cluster=cluster,
+            method=HttpMethod.PUT,
+            path=path,
+            headers=headers,
+            data=json.dumps(payload),
+            expected_status_code=200
+        )
+
+        logger.info(f"Successfully disabled bloom filter on index: {index_name}. Response: {response}")
+
+    def refresh(self, cluster: Cluster, index_name: str):
```

Comment:
Thanks for adding this, please update `check_doc_counts_match` to use this refresh definition